src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -16,14 +16,14 @@ You can also reach the moderators of this BEP through our [main discussion forum
 Most core principles of the original BIDS and particulars of BIDS-iEEG specification are adopted
 for this modality as well, though some special considerations and additional fields were added.
 
-Several [example x datasets](https://bids-standard.github.io/bids-examples/#ephys)
+Several [example x datasets](https://bids-standard.github.io/bids-examples/#microephys)
 have been formatted using this specification and can be used for practical guidance when curating a new dataset.
 
 ## Primary data file formats
 
-Unprocessed (animal) `ephys` data must be stored in an [open file format](https://en.wikipedia.org/wiki/Open_format),
+Unprocessed microelectrode electrophysiology (`icephys` and `ecephys` modalities) data must be stored in an [open file format](https://en.wikipedia.org/wiki/Open_format),
 while the native format, if different, can be stored in an optional  `sourcedata/` directory.
-The native file format is used in case conversion elicits the loss of crucial metadata specific to manufacturers and specific `ephys` systems.
+The native file format is used in case conversion elicits the loss of crucial metadata specific to manufacturers and specific hardware systems.
```

Comment:
"specific acquisition systems" ?

Reply:
```suggestion
The native file format is used in case conversion elicits the loss of crucial metadata specific to manufacturers and specific acquisition systems.
```